core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -602,10 +609,16 @@ private DefaultFunctionResolver utc_timestamp() {
    */
   private DefaultFunctionResolver week(BuiltinFunctionName week) {
     return define(week.getName(),
+        implWithProperties((functionProperties, arg) -> DateTimeFunction.weekOfYearToday(
+            DEFAULT_WEEK_OF_YEAR_MODE,
+            functionProperties.getQueryStartClock()), INTEGER, TIME),
```

Comment:
```suggestion
        implWithProperties(nullMissingHandlingWithProperties((functionProperties, arg) -> DateTimeFunction.weekOfYearToday(
            DEFAULT_WEEK_OF_YEAR_MODE,
            functionProperties.getQueryStartClock())), INTEGER, TIME),
```

Reply:
Fixed in 6465ec867cb5a13d21058604391687fe7cc03cb5